src/Root.tsx

```diff
@@ -191,7 +43,7 @@ export const Root: FC = () => {
         </div>
       </Navbar>
       <div className="flex h-full overflow-hidden bg-gray-50 dark:bg-gray-900">
-        <Sidebar collapsed={collapsed} itemsGroups={itemsGroups} />
+        <Sidebar collapsed={collapsed} itemsGroups={routes} />
```

Comment:
I think instead of having a two dimensions array just to satisfy this prop, we can have only one dimension array and use it as `itemGroups={[routes]}` in here.